### PR TITLE
Store NSIS resource in F3D Superbuild directly

### PR DESCRIPTION
**Pull request by mwestphal**
Related to: https://github.com/f3d-app/f3d/pull/700

This duplicate logo.ico between f3d and f3d-superbuild.

### CODE REVIEW

File: projects/win32/f3d.bundle.cmake [hunk 1]
```diff
@@ -67,6 +67,13 @@ install(
 
 ## NSIS Generator specific
 if (cpack_generator MATCHES "NSIS")
+
+  # NSIS Resources
+  file(COPY
+      "${superbuild_source_directory}/resources/logo.ico"
```
Comment (Meakk) [inline projects/win32/f3d.bundle.cmake:73]: can't we copy the ones located in f3d source?

Comment (mwestphal) [reply projects/win32/f3d.bundle.cmake:73]: we have no way to access the source file of a project apart from guessing it from the install location, which would be a very bad idea imo.

Comment (Meakk) [reply projects/win32/f3d.bundle.cmake:73]: Ok. I think `logotype64.bmp` can be removed from f3d then since it's only used for NSIS.

**Review by Meakk — APPROVED** [review]

File: projects/win32/f3d.bundle.cmake (hunk 1 above)
Comment (mwestphal) [reply projects/win32/f3d.bundle.cmake:73]: done in f3d-app/f3d#700

**Pull request merged**